os/adding-swap.md

```diff
@@ -0,0 +1,51 @@
+## Setup a swap file in the stateful partition
+
```

Comment:
It would be great to cover a use-case or two for this, explain why its off by default, etc.